create-ruler-guides/manifest.json

```diff
@@ -4,7 +4,6 @@
     "code": "plugin.js",
     "description": "This is a create ruler guides plugin ",
     "permissions": [
-      "content:read",
```

Comment:
needed by  penpot.getPage();

Reply:
getPage is outdated, i'll replace it with currentPage that doesn't need content:read